alpha/apps/kaltura/lib/kMrssManager.php

```diff
@@ -365,11 +273,21 @@ protected static function appendThumbAssetMrss(thumbAsset $thumbAsset, SimpleXML
 	 */
 	protected static function appendFlavorAssetMrss(flavorAsset $flavorAsset, SimpleXMLElement $mrss = null, kMrssParameters $mrssParams = null)
 	{
+		$servePlayManifest = false;
+		$playManifestClientTag = null;
+		$storageId = null;
+
 		if(!$mrss)
 			$mrss = new SimpleXMLElement('<item/>');
-		
+		else
+		{
+			$servePlayManifest = $mrss->getServePlayManifest();
```

Comment:
this section seems wrong, all these properties need to come from $mrssParams, not $mrss